### DISCUSSION TITLE
umi有没有什么插件能够把国外的股票嵌套到自己网站上

### DISCUSSION THREAD

**Discussion by liushunli** [umijs/umi#11728] [Q&A] [answered]
![image](https://github.com/umijs/umi/assets/12014406/7a3d5df5-8049-4d10-ae35-19594a2c1c4b)

就是类似新浪这种，有没有获取数据的api

**Comment by fz6m** [comment, marked as answer]
如果你要做一个大盘的视图，最合理、合规的方式是你们买一个证券数据库然后后端 api 下发给前端去自己绘图展示，如果要简单一点就考虑从第三方买数据 api 吧，应该有第三方代理商会售卖这种证券数据的 api 和现成的展示组件，需要注意如果从国外购买产品需要考虑到网络速度问题，具体我不清楚，自己研究下吧。
